### PR TITLE
remove rotation of keys endpoint from ux-backend

### PR DESCRIPTION
**Pull request by leelavg**
existing mechanism is only working as a proxy for deletion of secret w/ no added value, so deleting this endpoint and UI will be wired to directly perform the operator.

ref: #2469

### CODE REVIEW

File: services/ux-backend/handlers/onboardingtokens/handler.go [hunk 1]
```diff
@@ -5,12 +5,12 @@ import (
 	"net/http"
 
 	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
-	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
 	"k8s.io/klog/v2"
 )
 
 const (
 	onboardingPrivateKeyFilePath = "/etc/private-key/key"
+	ContentTypeTextPlain         = "text/plain"
```
Comment (nb-ohad) [inline services/ux-backend/handlers/onboardingtokens/handler.go:13]: The moving of the content type into a common handlers package made sense even without the new handler. Please do not undo it.

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: services/ux-backend/handlers/onboardingtokens/handler.go (hunk 1 above)
Comment (leelavg) [reply services/ux-backend/handlers/onboardingtokens/handler.go]: done.

**Comment by leelavg** [comment]
/override ci/prow/ocs-operator-bundle-e2e-aws

```
[1782237453432655872](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/red-hat-storage_ocs-operator/2569/pull-ci-red-hat-storage-ocs-operator-main-ocs-operator-bundle-e2e-aws/1782237453432655872)	#[2569](https://github.com/red-hat-storage/ocs-operator/pull/2569)
([a209a03](https://github.com/red-hat-storage/ocs-operator/pull/2569/commits/a209a0351ce77fe9aaafede17706752a44869e82)) by [leelavg](https://github.com/leelavg)	
08:08:21
1h33m37s	SUCCESS
```

**Pull request merged**